src/lib/core/datetime/native-date-adapter.ts

```diff
@@ -154,6 +162,12 @@ export class NativeDateAdapter extends DateAdapter<Date> {
 
   format(date: Date, displayFormat: Object): string {
     if (SUPPORTS_INTL_API) {
+      if (this.useUtcForDisplay) {
+        date = new Date(Date.UTC(
+            date.getFullYear(), date.getMonth(), date.getDate(), date.getHours(),
+            date.getMinutes(), date.getSeconds(), date.getMilliseconds()));
+        displayFormat = Object.assign({}, displayFormat, {timeZone: 'utc'});
```

Comment:
`Object.assign` is breaking in Google. Can you switch to our util `extendObject`?